### DISCUSSION TITLE
Link from website differ from reality

### DISCUSSION THREAD

**Discussion by michele-bettin** [Megabit/Blazorise#5045] [General]
The link component that I can see from the official website differ from the one on my project. My doesn't have the pretty animation that I can see on the website... Does someone has experienced something similar?

**Comment by stsrki** [comment]
We just have a little extra CSS targeting `a` element on our main web.

![image](https://github.com/Megabit/Blazorise/assets/900302/5459583e-66c1-439e-b638-c81a6ad82d3c)

**Reply by michele-bettin** [reply]
Thank you so much!!!
